tests/clients/test_logging.py

```diff
@@ -79,7 +79,7 @@ def test_parameters(self) -> None:
         assert spans[0].data["log"].get("parameters") is not None
 
     def test_no_root_exit_span(self) -> None:
-        agent.options.allow_exit_as_root = True
+        agent.options.allow_exit_as_root = False
```

Comment:
I understand the confusion caused by the same testcase name as `grpcio` 😅 but the usecase here is to not record the span even though `agent.options.allow_exit_as_root` is set to `True` since its an `INFO` log.

Reply:
OK. As a suggestion, it would be good to add additional comments for different behaviors or make explicit what the goal of the tests is in the method name.